allensdk/test/brain_observatory/behavior/behavior_project_cache/test_behavior_project_metadata_writer.py

```diff
@@ -29,6 +29,7 @@ def convert_strings_to_lists(df, is_session=True):
             .apply(lambda x: literal_eval(x))
 
 
+@pytest.mark.xfail
```

Comment:
Please include a reason, also consider squashing all these commits

Reply:
Oh yeah I'm definitely going to squash these